packages/nextjs/pages/lists/[listId].tsx

```diff
@@ -87,7 +92,7 @@ const ListDetail: NextPage<Props> = ({ list }) => {
 
         <div className="mt-16">
           <SharedProjects list={list} />
-          <SuggestProjects />
+          <SuggestProjects suggestedProjects={suggestedProjects} />
```

Comment:
I think that is better to use 

<SuggestProjects projectId={projectId} /> 

And then inside the SuggestProjects component, the useSuggestedProjects hook is used to get the suggested projects.